frontend/src/metabase-types/api/dashboard.ts

```diff
@@ -18,7 +18,7 @@ export interface Dashboard {
   name: string;
   description: string | null;
   model?: string;
-  ordered_cards: (DashboardOrderedCard | ActionDashboardCard)[];
+  dashcards: (DashboardOrderedCard | ActionDashboardCard)[];
   ordered_tabs?: DashboardOrderedTab[];
```

Comment:
Is there a reason for `ordered_tabs` to have `ordered_` prefix?
It looks like it falls into the same category as `ordered_cards`.
Perhaps it's a good candidate for a subsequent PR.

Reply:
I'm guessing only as a mirror. Aside from the fact that a sort may or may not have occurred somewhere that might give the cards some semblance of order, there is not reason to use `ordered_` or `Ordered` anywhere. The data structure is certainly not a sorted collection.